apps/web/components/common/Sidebar/index.tsx

```diff
@@ -0,0 +1,51 @@
+"use client";
+
+import React from "react";
+import { DoubleChevron } from "@ui/public";
+
+interface SidebarProps {
+  isOpen: boolean;
+  onClose: () => void;
+}
+
+/**
+ * 사이드바 컴포넌트입니다.
+ *
+ * @param props - 컴포넌트의 props
+ * @returns JSX.Element 사이드바 컴포넌트 요소
+ * 
+ * const [isSidebarOpen, setIsSidebarOpen] = useState(false);
+ *
+      \<Sidebar
+        isOpen=\{isSidebarOpen\}
+        onClose=\{() =\> \{
+          setIsSidebarOpen(false);
+        \}\}
+      /\>
+ * 
+ */
+
+function Sidebar(props: SidebarProps): JSX.Element {
```

Comment:
```suggestion
export default function Sidebar(props: SidebarProps): JSX.Element {
```
사소하지만 여기도 export style이 조금 벗어나있는거같아~